constants.go

```diff
@@ -161,7 +161,8 @@ func getEscapeToGroundBytes() []byte {
 }
 
 func getExecuteBytes() []byte {
-	executeBytes := getByteRange(0x00, 0x17)
+	executeBytes := getByteRange(0x00, 0x0F)
+	executeBytes = append(executeBytes, getByteRange(0x0A, 0x012)...)
```

Comment:
Should we comment on these ranges and why we've elected to skip values? I am thinking about the next person who  tries to maintain this code. Thanks.